### ISSUE TITLE
protege hangs on mac when opening empty owl file 

### ISSUE THREAD

**Issue by petacube** [protegeproject/protege#787]
steps to reproduce
1. create empty owl file with protege
2. save the file
3. close file
4. open file with protege - hangs 

able to reliably replicate
details of environment
mac os high sierra
16gb of ram

**Comment by petacube** [comment]
this is briliant tool. unfortunetely its unreliabilty, crashes in the middle of editing prevents me from using it. does anybody else experience the same issue?

**Comment by matthewhorridge** [comment]
@petacube please can you perform a quick test?  

Save the empty file in a new folder that does not have any other folders or files in it and see if the same problem occurs.

**Comment by petacube** [comment]
@matthewhorridge yes with i save file into folder when there is no other files hanging behavior not observed.

**Comment by matthewhorridge** [comment]
Please can you use this as a workaround.  

This issue comes from Protege scanning the directory structure to build a catalog file.
